eZ/Publish/Core/settings/containerBuilder.php

```diff
@@ -49,6 +43,25 @@
 $loader->load('utils.yml');
 $loader->load('tests/common.yml');
 
+// Cache settings (takes same env variables as ezplatform does, only supports "singleredis" setup)
+if (getenv('CUSTOM_CACHE_POOL') === 'singleredis') {
```

Comment:
Isn't it a bit awkward that you have the set the CUSTOM_CACHE_POOL env variable in order to get redis working?
Shouldn't it be sufficient to set the cache_pool parameter?

Or is this executed in a early stage where cache_pool parameter is yet not set ?

Reply:
@vidarl `containerBuilder.php` is used only for integration tests, to override default persistence cache.

Reply:
☝️  what he said